app/views/components/input_checkbox.scala.html

```diff
@@ -0,0 +1,28 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(
+    field: Field,
+    label: String
+)(implicit messages: Messages)
+
+
+<div class="form-group">
+        <div class="multiple-choice">
+            <input id="waste-type-1" name="waste-types" type="checkbox" value="waste-animal">
```

Comment:
Change:
`id="waste-type-1"` -> `id=@{field.id}`
`name="waste-types"` -> `name=@{field.name}`
`value="waste-animal"` -> `value=@{field.value}`

Put those things ^^ in new lines